.kokoro/run/common.cfg

```diff
@@ -0,0 +1,22 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Download trampoline resources. These will be in ${KOKORO_GFILE_DIR}
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Download secrets from Cloud Storage.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/nodejs-docs-samples"
+
+# All builds use the trampoline script to run in docker.
+build_file: "nodejs-docs-samples/.kokoro/trampoline.sh"
+
+# Tell the trampoline which build file to use.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-docs-samples/.kokoro/build-with-run.sh"
```

Comment:
Created this file on the assumption I need it to override the build script to `build-with-run.sh`

Reply:
Yep - you are correct. 🙂